html/index.html

```diff
@@ -3,11 +3,80 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Social media application</title>
-    <link rel="stylesheet" href="styles.css">
-    <script src="script.js" defer></script>
+    <title>Social media application home site</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" href="dist/css/index.css">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <link href="https://getbootstrap.com/docs/5.3/assets/css/docs.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css"></script>
 </head>
-<body>
+<body class="p-3 m-4 border-0 bd-example bg-light">
+    <ul class="nav justify-content-end">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="/html/index.html">KIT</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="feed/index.html">My feed</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="profile/index.html">My profile</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled">log out</a>
+        </li>
+        <li>
+          <form class="d-flex" role="search">
+            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-primary" type="submit">Search</button>
+          </form>
+        </li>
+      </ul>
+
+ 
+
+<!--this part needs to be nested in a card!-->
+
+    <div class="container my-5">
+      <div class="row justify-content-center align-items-center">
+        <div class="col-md-6">
+          <div class="card">
+            <div class="card-header">
+              <h1>Keep In Touch</h1>
+            </div>
+            <div class="card-body">
+              <form>
```

Comment:
This form should have an action attribute like `action="/profile"`. Read the brief of the CA again

Reply:
I see the mistake, the action atribute is now added to the form- no other js is added since the CA did not request js.